apps/L1G/restaurant/restaurant-frontend/src/components/auth/update-user-profile/EditEmailDialog.tsx

```
@@ -0,0 +1,78 @@
+    defaultValues: {
+      email: currentEmail,
+    },
+  });
+
+  const onSubmit = async (data: EmailFormValues) => {
+    try {
+      await onUpdate(data.email);
+      setOpen(false);
+      form.reset({ email: data.email });
+    } catch (error) {
+    } finally {
```

ESLint

[error] Empty block statement.

(no-empty)